alica_engine/include/engine/IAlicaTrace.h

```diff
@@ -34,8 +35,7 @@ class IAlicaTrace
         {
         }
 
-    private:
```

Comment:
why is this public now?

Reply:
Every tracing library uses their own internal variant definition, and to convert we usually need to call std::visit on the raw variant, so pretty much any tracing implementation will need access to the raw variant here.  For example to convert to AttributeValue from https://github.com/open-telemetry/opentelemetry-cpp/blob/main/api/include/opentelemetry/common/attribute_value.h

Reply:
I suppose we can remove the "friend IAlicaTrace" now too